source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -1847,20 +1847,12 @@ namespace AdaptiveNamespace
         ABI::AdaptiveNamespace::ActionType actionType;
         RETURN_IF_FAILED(action->get_ActionType(&actionType));
 
-        // now construct an appropriate button for the action type
```

Comment:
Do we actually have a unit test that you added or had failed earlier ? Or are we checking for no regressions wiht the existing unit tests at this point? Latter if fine (probably update teh HowVerified to clarify on the unit test bit)

Reply:
I just ran existing tests. Unfortunately, we don't have a great way to perform a unit test for this type of change at the moment.